app/views/users/show.html.erb

```diff
@@ -1,2 +1,43 @@
-<h1>Specific User details</h1>
-<p>Specific user's details will appear here</p>
+<h1>User details</h1>
+<h3 class="user-id">User ID: <%= @user.id %></h3>
+<main id="user-index" class="page">
+<div id="user-page">
+<div class="back-link-container">
+        <%= link_to "Home", root_path, class: "back-link" %>
+        <%= link_to "Back", :back, class: "back-link" %>
+      </div>
+      <br>
+      <div class="user-container">
+      <div class="image-div">
+<%= image_tag('user.png',  :width => "100px", :heigth=>"100px") %>
+</div>
+<div class="userdetails-div">
+<h3><%= @user.name %></h3>
+<p class="posts-counter">Number of posts: <%= @user.posts_counter%></p>
+</div>
+</div>
+<article class="user-bio">
+      <h3>Bio</h3>
+      <div class="bio"><%= @user.bio %></div>
+    </article>
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thanks for your great suggestion. Sure I will implement this a bit later on as this is also optional. as for now I need to work on next Project for this week which is adding Forms.